metals/src/main/scala/scala/meta/internal/metals/Doctor.scala

```diff
@@ -63,10 +64,14 @@ final class Doctor(
       onServer: MetalsHttpServer => Unit
   ): Unit = {
     if (config.executeClientCommand.isOn) {
-      val html = buildTargetsHtml()
+      val output = config.doctorFormat match {
```

Comment:
nit: can we use if/else instead of pattern matching here?

Reply:
Sure. I originally was also going to do Markdown, which led me down the match route, but then decided against it. I'll switch this.